test/server/clientlib.py

```diff
@@ -227,11 +231,18 @@ def make_random_id():
 
 
 @contextlib.contextmanager
-def screenshots_session(backend=None):
+def screenshots_session(backend=None, hasAccount=None):
     if backend:
-        user = ScreenshotsClient(backend=backend)
+        if hasAccount:
+            user = ScreenshotsClient(backend=backend, hasAccount=True)
+        else:
+            user = ScreenshotsClient(backend=backend)
     else:
-        user = ScreenshotsClient()
+        if hasAccount:
+            user = ScreenshotsClient(hasAccount=True)
+        else:
+            user = ScreenshotsClient()
```

Comment:
had to break it into if else to make python happy, in ideal world  user = ScreenshotsClient(backend=backend, hasAccount=hasAccount) should work

Reply:
That's weird, `ScreenshotsClient(backend=backend, hasAccount=hasAccount)` should work fine...?

Another option would be:

```py
def screenshots_session(**kw):
    user = ScreenshotsClient(**kw)
```

Reply:
Using kwargs works and simplifies it, updated PR